src/styles/utils/reset.css

```
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+p {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+}
+
+h3 {
+  margin: 0;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+}
```

You can combine these selectors
```suggestion
h2,
h3 {
  margin: 0;
}
```